src/screens/FeedScreen/RenderList.js

```diff
@@ -213,7 +225,8 @@ const RenderListFeed = (props) => {
             statusVote={voteStatus}
             showScoreButton={showScoreButton}
             onPressScore={() => showScoreAlertDialog(item)}
-            isSelf={item.anonimity ? false : selfUserId === item.actor.id}
+            isSelf={isSelf}
+            isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isSelf` prop is passed to the `Footer` component. This seems to be used for checking if the current user is the author of the post. If this prop is properly used in the `Footer` component, then this change is good.
```diff
+             isSelf={isSelf}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->